entries/entry03.md

```diff
@@ -1,7 +1,35 @@
 # Entry 3
```

Comment:
Sources: 0
EDP: 1
Knowledge: 0 (code snippets? evidence of learning?)
Skills: 1
Professionalism: 1